Documentation/Architecture/SpatialAwareness/SpatialAwarenessSystemArchitecture.md

```diff
@@ -2,42 +2,68 @@
+- [SpatialAwarenessMeshLevelOfDetail](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.SpatialAwarenessMeshLevelOfDetail)
+- [SpatialAwarenessSurfaceTypes](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.SpatialAwarenessSurfaceTypes)
 
-## Event Data Types
 
-- [MixedRealitySpatialAwarenessEventData](MixedRealitySpatialAwarenessEventData.md)
+## Events and Event Data
 
-## Configuration Profile
+- [IMixedRealitySpatialAwarenessObservationHandler<T>](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.IMixedRealitySpatialAwarenessObservationHandler-1)
+- [MixedRealitySpatialAwarenessEventData](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessEventData)
+- [MixedRealitySpatialAwarenessEventData<T>](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessEventData-1)
+
+## Profiles
+
+- [MixedRealitySpatialAwarenessSystemProfile](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessSystemProfile)
+- [BaseSpatialAwarenessObserverProfile](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.BaseSpatialAwarenessObserverProfile)
+- [MixedRealitySpatialAwarenessMeshObserverProfile](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessMeshObserverProfile)
 
-- [MixedRealitySpatialAwarenessProfile](MixedRealitySpatialAwarenessProfile.md)
-- [MixedRealitySpatialAwarenessProfileInspector](MixedRealitySpatialAwarenessProfileInspector.md)
- 
 ## Example Implementation
 
-The Mixed Reality Toolkit's default implementation of the spatial awareness system can be found in the Assets/MixedRealityToolkit/SDK/Features/SpatialAwarenessSystem folder. This implementation can be used as an example of how to build your own spatial awareness system. It is well documented and demonstrates each of the features described in this architecture specification.
+### Service
+
+The default spatial awareness system is a good example for developers looking to create their own implementation or to learn more about how the system works. It is provided as part of the Mixed Reality Toolkit Spatial Awareness service package. 
+
+The latest released code is available on [GitHub](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_development_hl2/Assets/MixedRealityToolkit.Services/SpatialAwarenessSystem/MixedRealitySpatialAwarenessSystem.cs).
+
+#### Class Documentation
+
+- [MixedRealitySpatialAwarenessSystem](xref:Microsoft.MixedReality.Toolkit.SpatialAwareness.MixedRealitySpatialAwarenessSystem)
+
+
+### Mesh Observer
+
+The Windows Mixed Reality Spatial Mesh Observer is a great example of a spatial  observer and is included as part of the Mixed Reality Toolkit Windows Mixed Reality provider package. 
+
+The latest released code is available on [GitHub](https://github.com/Microsoft/MixedRealityToolkit-Unity/tree/mrtk_release/Assets/MixedRealityToolkit.Providers/WindowsMixedReality).
```

Comment:
This doesn't link to the direct file (the one above does however - however the one above also actually points to mrtk_development_hl2, which it probably shouldn't).

Maybe just point to mrtk_development for both?

I did some digging and it looks like:

1) It's not possible to reference a true github root.
2) It's not possible for markdown to render differently based on the branch it's in (i.e. having some macro that is replaced with the current branch).

Reply:
hmmm. good catch. i'll take a look

Reply:
now pointing to the correct branch